### PR TITLE
Keep cluster up for debugging

### PR DESCRIPTION
**Pull request by ostromart**

### CODE REVIEW

File: jobs/config.json [hunk 1]
```diff
@@ -11939,6 +11939,7 @@
       "--check-leaked-resources",
       "--cluster=",
       "--deployment=gke",
+      "—-down=false",
```
Comment (BenTheElder) [inline jobs/config.json]: looks like you got a bit of unicode in here `—-` should be `--`

Comment (krzyzacy) [reply jobs/config.json]: lol probably because that’s my stroke input

**Comment by BenTheElder** [comment]
config_sort

File: jobs/config.json (hunk 1 above)
Comment (ostromart) [reply jobs/config.json]: oops, i copied that directly without looking closely. fixed now.

**Comment by BenTheElder** [comment]
`/hold cancel` anytime.  I think once the next job starts it will clean up the previous cluster before bringing up a new one..

**Comment by krzyzacy** [comment]
>  I think once the next job starts it will clean up the previous cluster before bringing up a new one..

not really, it has `--cluster=` flag

**Pull request merged**